cli/lib/models/manufactured_class.dart

```diff
@@ -0,0 +1,21 @@
+import 'item_class.dart';
+
+class ManufacturedItem extends ItemsClass {
+  double itemTax = 0.0;
+  double itemFinalPrice = 0.0;
+  ManufacturedItem(String? name, String? type, int quantity, double price)
+      : super(name, type, quantity, price);
+  @override
+  double calculateTax() {
+    double tax = 12.5 / 100 * super.itemPrice;
+    tax += 2.0 / 100 * (super.itemPrice + tax);
+    itemTax = tax;
+    return itemTax;
+  }
+
+  @override
+  double calculatePrice() {
+    itemFinalPrice = super.itemPrice + itemTax;
+    return itemFinalPrice;
+  }
+}
```

Comment:
Convert all the hard-coded values like 12.5, 2.0 etc into constants and give them meaningful names so that it is understandable what they represent